### PR TITLE
Added Analytics feature

### PR DESCRIPTION
**Pull request by Dhruv88**
Solves #17 
Keeps track of per day visits to URL from the day of creation to expiry.
Also an API to retrieve this count for a particular URL has been added.

### CODE REVIEW

**Comment by shashwatj07** [comment]
resolve conflicts

**Comment by Dhruv88** [comment]
> resolve conflicts

Done

**Comment by shashwatj07** [comment]
We should have a way of handling cases where the URL has never been used.

File: README.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ curl http://localhost:8080/auth/token -u username:password
 ```
 (Note: The validity period is 30 days for the bearer token generated)
 
+GET Request (to get analytics for a url):
+```
+curl http://localhost:8080/analytics/<random-hash|custom-alias> --include --header "Content-Type: application/json" --request "GET" -H "Authorization: Bearer <auth_token>"
+```
```
Comment (shashwatj07) [inline README.md:18]:
```
curl http://localhost:8080/analytics/<random-hash|custom-alias> -H "Authorization: Bearer <auth_token>"
```

**Review by shashwatj07 — CHANGES_REQUESTED** [review]
make this change

**Review by shashwatj07 — APPROVED** [review]

**Pull request merged**